driver/src/test/resources/org/kaazing/k3po/driver/internal/http/http.client.channel.abort/request.rpt

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright 2007-2015, Kaazing Corporation. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+accept tcp://localhost:8080
+accepted
+connected
+read "GET /path HTTP/1.1\r\n"
+read "Host: localhost:8080\r\n"
+read "\r\n"
+
+closed
```

Comment:
Abort is supposed to kill the transport beneath it.  So for tcp we have no API on that.  I didn't implement anything for it, and as of now if someone used it in a script it would have no effect.  I also did not implement anything for Agrona (File transport).  Not sure what abort would mean in that case either.